core/src/main/scala/com/salesforce/op/filters/RawFeatureFilter.scala

```diff
@@ -93,7 +94,8 @@ class RawFeatureFilter[T]
   val correlationType: CorrelationType = CorrelationType.Pearson,
   val jsDivergenceProtectedFeatures: Set[String] = Set.empty,
   val protectedFeatures: Set[String] = Set.empty,
-  val textBinsFormula: (Summary, Int) => Int = RawFeatureFilter.textBinsFormula
+  val textBinsFormula: (Summary, Int) => Int = RawFeatureFilter.textBinsFormula,
+  val timePeriod: Option[TimePeriod] = None
```

Comment:
docs for `timePeriod`

Reply:
also, please add `timePeriod` to `OpWorkflow.withRawFeatureFilter` as well

Reply:
done